design-documents/cart-discount-itemization.md

```diff
@@ -0,0 +1,58 @@
+**Cart Discount Itemization**
+
+Overview
+
+The individual itemized break down of the discounts applied at cart and cart item level are to be computed and persisted on every order.
+
+**Existing Implementation**
+
+The itemized discounts are computed and stored with extension attributes on 
+Magento\Quote\Api\Data\CartItemInterface 
+Magento\Quote\Api\Data\AddressInterface. 
```

Comment:
can you please specify an interface of extension?

Reply:
Updated the doc.